src/app/organizations/collection/collection.component.html

```diff
@@ -133,6 +133,7 @@ <h5>
                       </span>
                     </div>
                   </div>
+                  <span>{{ entry.topic }}</span>
```

Comment:
On further reflection, shouldn't this be causing a build error until dockstore/dockstore#5980 is merged?

If so, the dependency might be worth clarifying in the PR description.

Reply:
> On further reflection, shouldn't this be causing a build error until [dockstore/dockstore#5980](https://github.com/dockstore/dockstore/pull/5980) is merged?
> 
> If so, the dependency might be worth clarifying in the PR description.

Would be great if there was a lightweight way to point the PR at a version of the webservice that supported the new `topic` field, so that reviewers could easily do things like check out the branch and test it.

Reply:
Think we have part of this with https://github.com/dockstore/dockstore-ui2?tab=readme-ov-file#build pointing at branches or commits. I think the next part would be to upload artifacts to artifactory from branches other than develop